ReactWindows/ReactNative/Modules/Storage/AsyncStorageModule.cs

```diff
@@ -20,5 +39,240 @@ public override string Name
                 return "AsyncLocalStorage";
             }
         }
+
+        /// <summary>
+        /// Given an array of keys, this returns a map of (key, value) pairs for the keys found, and
+        /// (key, null) for the keys that haven't been found.
+        /// </summary>
+        /// <param name="keys">Array of key values</param>
+        /// <param name="callback">Callback.</param>
+        [ReactMethod]
+        public void multiGet(JArray keys, ICallback callback)
+        {
+            if (keys == null)
+                throw new ArgumentNullException(nameof(keys));
+
+            if (callback == null)
+                throw new ArgumentNullException(nameof(callback));
```

Comment:
Generally speaking, it should not be possible for a callback to be null.  We could probably live with a Debug.Assert here.